### PR TITLE
refactor: move material composition code to the left

### PR DESCRIPTION
**Pull request by andiwand**
early exit and whitespace cleanup

picked up on this file because it crashed for me when `A >= 100`

changes are not critical so we can also drop it

### CODE REVIEW

File: Examples/Scripts/MaterialMapping/materialComposition.C [hunk 1]
```diff
@@ -108,128 +108,131 @@ void materialComposition(const std::string& inFile, const std::string& treeName,
...
+  // Draw all the atomic elements & get the histogram
+  inputTree->Draw("mat_A>>hA(100,0.5,100.5)");
+  TH1F* histA = dynamic_cast<TH1F*>(gDirectory->Get("hA"));
+  histA->Draw();
+
+  auto outputFile = TFile::Open(outFile.c_str(), "recreate");
+
+  float v_eta = 0;
+  float v_phi = 0;
+  std::vector<float>* stepLength = new std::vector<float>;
```
Comment (benjaminhuth) [inline Examples/Scripts/MaterialMapping/materialComposition.C:128]: Are those vectors ever properly destructed? However, since this is not really part of the PR, maybe not that important...

Comment (andiwand) [reply Examples/Scripts/MaterialMapping/materialComposition.C:128]: I fear not and there seem to be a few places in this code were we do not care about deallocation :skull: I would leave it as it is for now

**Review by benjaminhuth — APPROVED** [review]
